### ISSUE TITLE
Logo upload not working with Cherry Framework theme 

### ISSUE THREAD

**Issue by aowens42** [Automattic/WP-Job-Manager#604]
I haven't found this addressed specifically, but we're trying to deploy the plugin that uses Cherry Framework and a child theme. If we can get this one thing fixed, we'll start with addons, including ecommerce.

The plugin works great except for the company logo on the jobs submission page. I'm guessing there is a conflict between the theme and/or another plugin and WP Job Manager.

I was careful to ensure nothing was surrounding the shortcode and even deployed it by itself on a blank test page to no avail.

When I try to upload a logo, it says nothing has been selected and then the preview button stops functioning.

Could you take a look at it? Here's a link:
http://crbjbizwire.com/post-a-job/

Thanks!
Andy

**Comment by aowens42** [comment]
I think I've narrowed it down to definitely a theme conflict.

We're not using any CSS overrides (just because we haven't had the need to yet), but I switched to the Twenty Fourteen theme, and the plugin worked perfectly, including the upload. Glad it's not a plugin conflict, since we have several that help automate a large email send each day.

Wonder if I can CSS this thing into working with our Cherry Framework theme. That's my hope!

Thanks,
Andy
